app/features/configuration/configuration_controller.rb

```diff
@@ -63,8 +63,8 @@ def keys
     # @return [Set[String]]
     def post_parameter_list_for_validation
       return Set.new(
-        %w(encryption_key ci_user_email ci_user_password repo_url
-           clone_user_email clone_user_api_token)
+        %w(encryption_key ci_user_email ci_user_password ci_user_api_token
+           repo_url clone_user_email clone_user_api_token)
```

Comment:
maybe we need to rename things, like `clone_user_email` should actually be the bot now, iff we do the invitation dance

Reply:
So, I'm not sure where we actually use the `clone_user_email`. I was looking through [the code where we pre-populate the email](https://github.com/fastlane/ci/blob/master/app/features/login/login_controller.rb#L39) - but we actually get that email from the API token itself. Should we just remove the `*_user_email` environment variables?

I can't see where they're being used elsewhere.